.github/PULL_REQUEST_TEMPLATE/release_pull_request.md

```diff
@@ -22,17 +22,15 @@ No extra PRs yet. 🎉
 ## Test plan
 
 - Use the main WP apps to test the changes above. 
-- Check WPAndroid and WPiOS PRs if there are specific tests to run.
 - Smoke test the main WP apps for [general writing flow](https://github.com/wordpress-mobile/test-cases/tree/master/test-cases/gutenberg/writing-flow).
 
 ## Release Submission Checklist
 
+- [ ] Approve and run optional Android and iOS UI tests
 - [ ] Release number was bumped in `package.json` and `gutenberg/packages/react-native-editor/package.json`.
 - [ ] Aztec dependencies are pointing to a stable release.
-  - iOS: 'grep WordPressAztec-iOS RNTAztecView.podspec'.
   - iOS: Aztec dependencies match in `RNTAztecView.podspec` and `gutenberg/packages/react-native-aztec/RNTAztecView.podspec`.
-  - Android: 'grep aztecVersion react-native-aztec/android/build.gradle'.
```

Comment:
@mchowning I think this check (and the iOS one) are now already being made by the automation script: https://github.com/wordpress-mobile/gutenberg-mobile/blob/c67e9c8ef557adbacd5ba191d659a7e722b7f691/bin/release_prechecks.sh#L51
Do you think it still makes sense to keep them in the checklist?

Reply:
👍 I think it makes sense to take them out. By the same argument, should we also take out "Release number was bumped in `package.json` and `gutenberg/packages/react-native-editor/package.json`."?

Reply:
Yes! That also makes sense, thanks for the suggestion 👍